tutorials/particle_trackpy.md

```diff
@@ -0,0 +1,4 @@
+# tracks layer tutorial
+
+To be completed (see
+[issue #88](https://github.com/napari/napari.github.io/issues/88)).
```

Comment:
I would really prefer we not put to-do notes in external, public facing documentation.

Reply:
Right, it was more for identifying why this document is here, but I'll make sure to remove.